### ISSUE TITLE
example data of what a "project" should contain for supabase

### ISSUE THREAD

**Issue by sparrowsl** [christex-foundation/pipeline#121]
to see the fields that a project contains, one will have to dig into the codebase (for hours?), watch the form for creating new project, etc...,
```js
const project = {
      email: "email@mail.com;
      portfolio: "portfolio link";
      github: "github link";
      // ... other fields
}
```

basically something like the above snippet in an example file or even in the CONTRIBUTING.md file will help a lot.

**Comment by sparrowsl** [comment]
also adding an example of the table names to add in supabase.
I noticed I don't know the name of the table to add, except watch some error messages which are not always reliable.

**Comment by sparrowsl** [comment]
closed when #181 is merged
